rust-toolchain.toml

```diff
@@ -1,3 +1,2 @@
 [toolchain]
-channel = "nightly-2023-08-01"
-
+channel = "nightly-2024-08-01"
```

Comment:
Any reason do thsi change?
Basically it should be as same as playground, and our playground is as same as zkwasm.
So we may not want to change it.